examples/leasing/node_release_script.py

```diff
@@ -0,0 +1,187 @@
+#! /bin/python
+"""
+This script will free nodes from a project after
+a specific amount of time.
+
+You can find more information including
+config file and status file format in
+README file.
+"""
+
+import time
+import ConfigParser
+
+from hil.client.client import Client, RequestsHTTPClient
+from hil.client.base import FailedAPICallException
+
+
+class HILClientFailure(Exception):
+    """Exception indicating that the HIL client failed"""
+
+
+class StatusFileError(Exception):
+    """Exception regarding Status File Format"""
```

Comment:
You raise these exceptions in a couple places, but never catch them; probably you should add another `except` clause at the bottom of the script that prints a human-friendly error when these get raised.